lisp/plugins/action_cues/collection_cue.py

```diff
@@ -140,8 +183,10 @@ def _showAddCueDialog(self):
 
     def _removeCurrentCue(self):
         row = self.collectionView.currentIndex().row()
-        cueId = self.collectionModel.rows[row][0]
+        if row == -1:
```

Comment:
This is not in the scope of this change, but I'll leave it here for now. Without the check, the application is crashing completely when "Remove" button is pressed without any selection. We might want to move that to a separate PR and make some UX improvements like disabling Remove button and only enabling it when something is selected.